google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurableTest.java

```diff
@@ -35,14 +35,16 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
 public class AppEngineCloudConfigurableTest extends PlatformTestCase {
   private AppEngineCloudConfigurable appEngineCloudConfigurable;
   private SystemEnvironmentProvider environmentProvider;
   private CloudSdkService cloudSdkService;
   private TextFieldWithBrowseButton cloudSdkDirectoryField;
 
-  private static final String CLOUD_SDK_DIR_PATH = new File("/a/b/c/gcloud-sdk").getAbsolutePath();
+  private static final Path CLOUD_SDK_DIR_PATH = Paths.get("/a/b/c/gcloud-sdk");
```

Comment:
I think Paths.get is not very smart about doing platform independent stuff. Instead I think its better to break it up yourself:

Paths.get("my", "path", "...")

Its just a unit test, so its not a huge deal unless we start running them on other platforms.